gpkit/nomials/math.py

```diff
@@ -388,6 +393,20 @@ def subinplace(self, substitutions):
         self.varkeys = KeySet(self.left.vks)
         self.varkeys.update(self.right.vks)
 
+    def relaxed(self, relaxvar):
```

Comment:
oh, interesting! why is this a method of a Posynomial, however? I think this should be a function that takes a posynomial as an argument, and it should be in `relax.py`

Reply:
I have thought about this. Let's discuss.

Reply:
I think we agreed this is good!